### PR TITLE
Clean up Messenger related code

### LINKED ISSUE #76: Clean up Messenger related code

### PR DESCRIPTION
**Pull request by WerLaj**
Fixes #76

### CODE REVIEW

File: moviebot/controller/server.py [hunk 1]
```diff
@@ -55,80 +54,12 @@ def set_webhook():
...
-    """Gets verify token from config file."""
-    path = "config/bot_token.yaml"
-    with open(path, "r") as file:
-        config = yaml.load(file, Loader=yaml.Loader)
-        VERIFY_TOKEN = config["MESSENGER_VERIFY_TOKEN"]
-        return VERIFY_TOKEN
-
-
-@app.route("/", methods=["GET", "POST"])
-def receive_message():
```
Comment (NoB0) [inline moviebot/controller/server.py:82]: Shouldn't this method stay for the Telegram controller?

File: moviebot/controller/server.py [hunk 2]
```diff
@@ -55,80 +54,12 @@ def set_webhook():
...
-
-
-def verify_fb_token(token_sent):
-    if token_sent == verify_token():
-        return request.args.get("hub.challenge")
-    return "Invalid verification token"
-
-
-def action(output):
-    """Gets user id and payload from output and runs get_message in the controller.
```
Comment (NoB0) [inline moviebot/controller/server.py:101]: Shouldn't this method adapted for Telegram?

File: moviebot/controller/server.py [hunk 3]
```diff
@@ -55,80 +54,12 @@ def set_webhook():
...
-    controller_messenger.initialize(user_id)
-    payload = get_message(output)
-    print(payload)
-    if payload is not None:
-        if controller_messenger.run_method(user_id, payload):
-            controller_messenger.send_message(user_id, payload)
-
-
-def get_message(output):
-    """Gets payload from output.
```
Comment (NoB0) [inline moviebot/controller/server.py:118]: Shouldn't this method adapted for Telegram?

**Review by NoB0 — COMMENTED** [review]
I am wondering if some methods should be adapted for Telegram instead of being remove. 
Other question regarding `run.py`, why restrict to `TelegramController`? One coudl think of using `ControllerTerminal`.

File: moviebot/controller/server.py (hunk 1 above)
Comment (WerLaj) [reply moviebot/controller/server.py:82]: According to the description, this method is for receiving Messenger POST requests.

File: moviebot/controller/server.py (hunk 2 above)
Comment (WerLaj) [reply moviebot/controller/server.py:101]: This method is used only in `receive_message()`

File: moviebot/controller/server.py (hunk 3 above)
Comment (WerLaj) [reply moviebot/controller/server.py:118]: This method is used only in `receive_message()`

**Comment by WerLaj** [comment]
> 

About `run.py`, there is still an option there to use `ControllerTerminal.`

**Review by NoB0 — APPROVED** [review]
LGTM.

**Review by IKostric — APPROVED** [review]
LGTM

**Pull request merged**